widget-reference/dropdown.md

```diff
@@ -40,9 +40,9 @@ Read more about submitting Input data to an API below
 | Property | Description |
 | :--- | :--- |
 | **Label** | Sets the label of the dropdown. |
-| **Selection Type** | Lets you control whether you want the dropdown to be a Single Select or a Multi Select dropdown. Single Select allows users to choose only one value from the list of options whereas Multi Select allows for selection of more than one value by the user. |
+| **Selection Type** | Lets you control whether you want the dropdown to be a Single Select or a Multi-Select dropdown. Single Select allows users to choose only one value from the list of options whereas Multi-Select allows for selection of more than one value by the user. |
```

Comment:
multi select is sometimes starting with caps and sometimes not. Lets standardise it